hovor/core.py

```diff
@@ -32,7 +33,7 @@ def run_interaction(configuration_provider):
 class ConversationLogInterface:
     def write_message(self, entity, message):
         pass
-    def write_diologue_pair(self, agent_message=None, user_message=None):
+    def write_diologue_pair(self, agent_message=None, user_message=None, action=None, action_type=None):
```

Comment:
Should be spelled dialogue (happens in a few other places too)

Reply:
Good point, done!